modules/oe_content_publication/config/install/core.entity_form_display.node.oe_publication.default.yml

```diff
@@ -96,37 +139,80 @@ content:
     region: content
   oe_publication_contacts:
     type: inline_entity_form_complex
-    weight: 9
+    weight: 12
     region: content
     settings:
       form_mode: default
       revision: true
-      label_singular: ''
-      label_plural: ''
+      override_labels: true
+      label_singular: contact
+      label_plural: contacts
       allow_new: true
       match_operator: CONTAINS
-      override_labels: false
       collapsible: false
       collapsed: false
       allow_existing: false
       allow_duplicate: false
     third_party_settings: {  }
+  oe_publication_country:
+    weight: 11
+    settings:
+      match_operator: CONTAINS
+      match_limit: 10
+      size: 60
+      placeholder: ''
+    third_party_settings: {  }
+    type: skos_concept_entity_reference_autocomplete
+    region: content
   oe_publication_date:
-    weight: 4
-    settings: {  }
+    weight: 6
+    settings:
+      date_order: DMY
+      time_type: none
+      increment: 15
+    third_party_settings: {  }
+    type: datetime_datelist
+    region: content
+  oe_publication_last_updated:
+    weight: 7
+    settings:
+      date_order: DMY
+      time_type: none
+      increment: 15
+    third_party_settings: {  }
+    type: datetime_datelist
+    region: content
+  oe_publication_thumbnail:
+    weight: 8
+    settings:
+      match_operator: CONTAINS
+      match_limit: 10
+      size: 60
+      placeholder: ''
     third_party_settings: {  }
-    type: datetime_default
+    type: entity_reference_autocomplete
     region: content
   oe_publication_type:
     type: skos_concept_entity_reference_autocomplete
-    weight: 6
+    weight: 0
```

Comment:
The resource type shouldn't be the first item in the form.

Reply:
No need to be very accurate with the form since we will change it in ewcms but lets at least be consistent and put the title first.

Reply:
I reordered the form fields.